build/dependencies.props

```diff
@@ -42,6 +42,7 @@
     <MicrosoftAspNetCoreWebUtilitiesPackageVersion>2.1.0-preview1-28055</MicrosoftAspNetCoreWebUtilitiesPackageVersion>
     <MicrosoftAspNetWebApiClientPackageVersion>5.2.4-preview1</MicrosoftAspNetWebApiClientPackageVersion>
     <MicrosoftCodeAnalysisCSharpPackageVersion>2.3.1</MicrosoftCodeAnalysisCSharpPackageVersion>
+    <MicrosoftCodeAnalysisCSharpWorkspacesPackageVersion>2.6.1</MicrosoftCodeAnalysisCSharpWorkspacesPackageVersion>
```

Comment:
This isn't right. This should be the same version as all of the other Roslyn packages.

I assume this is for tests right?